# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/ntp-in6.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ntp-4.2.0/ntpd/ntp_intres.c.orig	2003-07-17 12:27:28.000000000 +0200
+++ ntp-4.2.0/ntpd/ntp_intres.c	2004-11-23 10:45:08.499283176 +0100
@@ -396,6 +396,7 @@
 	register char *cp;
 	register struct conf_entry *ce;
 	unsigned int len;
+	struct in6_addr in6addr_any = IN6ADDR_ANY_INIT;
 
 #ifdef DEBUG
 	if (debug > 1)
